### ISSUE TITLE
Testing on Safari is not working in Catalina

### ISSUE THREAD

**Issue by lucacasonato** [dart-lang/test#1253]
In Catalina opening a file in Safari requires the user to confirm that they want to open the file in Safari. Because of this testing on safari does not work anymore without manual user interaction. According to my research this has been broken since at least Mojave.

**Comment by grouma** [comment]
You are correct as I verified this is an issue on Mojave. A quick Google search shows other testing frameworks also have this issue:
https://github.com/testem/testem/issues/1387

There is a possible workaround:

> I have ran into this as well with Github Actions VMs and locally as well. There's something for Karma via AppleScript to workaround this. https://github.com/brandonocasey/karma-safari-applescript-launcher.

That being said I don't think this is something we will make progress on in the near term.
